### DISCUSSION TITLE
我电脑上下了6,70个steam的游戏，嫌麻烦我全部打包分享出来可以不？

### DISCUSSION THREAD

**Discussion by ztkedou** [pjy612/SteamManifestCache#29] [Share]
code：A410FC3AF7F650E4946FB5EF456BD68CCEE15113
全部打包好了，4t的固态所以基本上会玩的都下下来了，一共6，70个游戏，一个个分享有点麻烦。不知道这样行不行。
邮箱是yuji123@163.com，github上面我好想也点了公开。

https://www.123pan.com/s/DTYTjv-kFVtH.html
![1](https://github.com/pjy612/SteamManifestCache/assets/32985948/34212e7e-6445-484b-ad74-7c83b08ace06)

**Comment by ztkedou** [comment]
顺便问一下有没有官方的tg群呀，tg我也经常刷。没看到哪里有群

**Reply by pjy612** [reply]
直接 steamtools 官方群，查收邮件

**Reply by ztkedou** [reply]
原来是官方人员！我在群里，多谢大佬！

**Reply by pjy612** [reply]
非官方，但是 在 tools 群，入群早 混了个脸熟，顺便兼职 群管
